test/yaml_writer_test.rb

```diff
@@ -23,34 +23,38 @@
 require_relative "spec_helper"
 require "storage"
 require "y2storage/yaml_writer"
+require "y2storage/disk"
+require "y2storage/partition_tables"
+require "y2storage/filesystems"
 
 describe Y2Storage::YamlWriter do
 
-  it "produces yaml of a simple disk and partition setup" do
+  let(:test_environment) do
+    Storage::Environment.new(true, Storage::ProbeMode_NONE, Storage::TargetMode_DIRECT)
+  end
+  let(:storage) { Storage::Storage.new(test_environment) }
```

Comment:
Yes, this is only the first part. I just meant to check this in before it becomes rotten and irrelevant again due to rebasing.